client/protocols/backup.py

```diff
@@ -176,4 +188,21 @@ def send_messages(self, access_token, messages, timeout):
         response = urllib.request.urlopen(request, serialized_data, timeout)
         return json.loads(response.read().decode('utf-8'))
 
+    @staticmethod
+    def cp(msg):
+        try:
+            # windows
+            if os.name == 'nt':
+                res = os.system('echo ' + msg.strip() + '| clip')
+            # unix
+            elif os.name == 'posix':
+                res = os.system('echo ' + msg + \
+                                ' | tr -d "\n" > /dev/null 2>&1' + \
+                                ' | pbcopy > /dev/null 2>&1')
+            else:
+                raise NotImplementedError('Unsupporeted OS')
```

Comment:
I don't think we should raise an error here. Let's log that it didn't work (`logging.debug`) and return something that indicates that it failed